### ISSUE TITLE
Polymer 1.0: FF: paper-slider dragging no working

### ISSUE THREAD

**Issue by karstor** [Polymer/paper-slider#47]
Polymer 1.0

On FF 38.01 on Win 7 64-bit, the dragging of the paper-slider does not work, but throws a JS error:
"event is not defined".

This is due to a typo in the function _onTrack in paper-slider.html (1.0 version from 2015). It currently reads:
 _onTrack: function(e) {
      switch (event.detail.state) {
        case 'end':
          this._trackEnd(event);
          break;
        case 'track':
          this._trackX(event);
          break;
        case 'start':
          this._trackStart(event);
          break;
      }
    },

BUT IT SHOULD READ

 _onTrack: function(e) {
      switch (e.detail.state) {
        case 'end':
          this._trackEnd(e);
          break;
        case 'track':
          this._trackX(e);
          break;
        case 'start':
          this._trackStart(e);
          break;
      }

changing "event" to "e" makes it work.

**Comment by hoony2520** [comment]
You are right. There are no 'event' variables inside '_onTrack' function. I think Polymer developers made a mistake. Once I changed this variable from 'event' to 'e', It worked perfectly. I appreciate it!

**Comment by karstor** [comment]
No problem. Glad it helped somebody!
